source-academy/frontend
# Disable remote execution tab if deployment is playground only

### Description

Resolves #2224, ~~allow disabling of the remote execution tab in playground configurable through the env file, under the newly added boolean value `REACT_APP_ENABLE_REMOTE_EXECUTION`.~~

~~The default value configured is `FALSE` (in the .env.example file)~~

The remote execution tab will only show when the `REACT_APP_PLAYGROUND_ONLY` is set to false (i.e. deployment with a backend), instead of the current behavior where it shows regardless of the deployment type.

### Type of change


### How to test

Pull and run the changes locally, and by default the remote execution tab should not be accessible anymore in the playground.

Code is tested working on latest version of Chrome 123.

### Checklist
